CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 ## Unreleased
 
-### Changes
+### Fixes
 
+- Fix caching issues (#815) @Tyrrrz
```

Comment:
In retrospect, the changelog entries are really not very helpful:

```suggestion
- Fix serialization of transactions when filesystem caching is enabled. (#815) @Tyrrrz
```

The one below? "fix tracing middleware"? What fix? Does it affect me? Should I upgrade?